File: patterns/Pattern3.java
import java.util.*;

class Pattern3{

	public static void main(String args[]){
		Scanner sc= new Scanner(System.in);
		int n;
		System.out.println("Enter a Range: ");
		n = sc.nextInt();
		for(int i=0; i<=n; i++)
		{
			for(int j=0; j<=i; j++)
			{
				System.out.print(" ");
			}
			for(int k=n; k>i; k--)
			{
				System.out.print("*");
			}
			System.out.println();
		}
	}
}

File: patterns/pattern_3.cpp
#include<iostream>
using namespace std;
int main()
{
	int n;
	cout <<"Enter a Range: ";
	cin>>n;
	for(int i=0; i<=n; i++)
	{
		for(int j=0; j<=i; j++)
		{
			cout <<" ";
		}
		for(int k=n; k>i; k--)
		{
			cout <<"*";
		}
		cout <<endl;
	}
	return 0;
}